src/java/com/zimbra/graphql/utilities/XMLDocumentUtilities.java

```diff
@@ -75,6 +77,26 @@ public static Element executeDocument(DocumentHandler handler, ZimbraSoapContext
         return handler.handle(request, context);
     }
 
+    /**
+     * Executes a given request on a document handler.
```

Comment:
should add more details in the description about how this one is different from other executeDocument.